db/thing.py

```diff
@@ -87,10 +96,23 @@ class Thing(Base, AutoBaseMixin, ReleaseMixin):
         )
     )
 
-    field_events = relationship(
+    # --- Relationships ---
+    # One-To-Many: A Thing can have many FieldEvents over time.
+    field_events: Mapped[List["FieldEvent"]] = relationship(
         "FieldEvent", back_populates="thing", cascade="all, delete-orphan", uselist=True
     )
 
+    # One-To-Many: A Thing can have many Deployments of sensors (equipment) over time.
+    deployments: Mapped[List["Deployment"]] = relationship(
+        "Deployment", back_populates="thing", cascade="all, delete-orphan"
+    )
+
+    # --- Association Proxies ---
+    # Proxy to directly access the Sensor deployed at this Thing.
+    sensor: AssociationProxy[List["Sensor"]] = association_proxy(
```

Comment:
should this be named `sensors` since multiple sensors may be associated with a `thing`?